### DISCUSSION TITLE
Symbols for Fortran code are not grouped correctly

### DISCUSSION THREAD

**Discussion by oxcrow** [hedyhli/outline.nvim#101] [Help]
Hello,

I observed that Fortran symbols are not grouped/indented correctly.

Data members inside derived types (i.e structs/classes)) are not grouped correctly in the outline, and neither are the modules or functions.

I'm using [Fortls LSP](https://github.com/fortran-lang/fortls) with Neovim.

Thanks!

![bug](https://github.com/hedyhli/outline.nvim/assets/172302111/20c59b65-dd37-458c-ac81-27fbf75864e3)

**Comment by hedyhli** [comment]
Hi, unfortunately I don't use Fortran and don't currently have the bandwidth to troubleshoot the issue. I will convert this to a discussion in case others have similar issues.
